### PR TITLE
Update the event stream section on the RFC30.

### PR DESCRIPTION
**Pull request by thomas-k-cameron**
## Motivation and Context
[Based on these comments](https://github.com/awslabs/smithy-rs/pull/2198#pullrequestreview-1260817164)

## Description
updates the event stream section on the RFC30.

## Testing
NA

## Checklist
NA
----

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: design/src/rfcs/rfc0030_serialization_and_deserialization.md [hunk 1]
```diff
@@ -206,53 +206,56 @@ The resulting size of the serialized data is smaller when tagged externally, as
...
+    pub acl: std::option::Option<crate::model::ObjectCannedAcl>,
+    pub body: aws_smithy_http::byte_stream::ByteStream,
+    // ... other fields
 }
 ```
 
+## SDK Data types to skip
+For SDK data types that includes `@streaming union` in one of their field, we do NOT implement `serde` traits.
+
+As of writing, following Rust data types corresponds to `@stremaing union`.
```
Comment (jdisanti) [inline design/src/rfcs/rfc0030_serialization_and_deserialization.md:245]:
```suggestion
As of writing, following Rust data types corresponds to `@streaming union`.
```

File: design/src/rfcs/rfc0030_serialization_and_deserialization.md [hunk 2]
```diff
@@ -206,53 +206,56 @@ The resulting size of the serialized data is smaller when tagged externally, as
...
-        crate::error::ExampleTranscriptResultStreamError,
-    >
+pub struct PutObjectInput {
+    pub acl: std::option::Option<crate::model::ObjectCannedAcl>,
+    pub body: aws_smithy_http::byte_stream::ByteStream,
+    // ... other fields
 }
 ```
 
+## SDK Data types to skip
```
Comment (jdisanti) [inline design/src/rfcs/rfc0030_serialization_and_deserialization.md:242]:
```suggestion
## Data types to exclude from ser/de code generation
```

File: design/src/rfcs/rfc0030_serialization_and_deserialization.md [hunk 3]
```diff
@@ -206,53 +206,56 @@ The resulting size of the serialized data is smaller when tagged externally, as
...
+For SDK data types that includes `@streaming union` in one of their field, we do NOT implement `serde` traits.
```
Comment (jdisanti) [inline design/src/rfcs/rfc0030_serialization_and_deserialization.md:243]:
```suggestion
For data types that include `@streaming union` in any of their fields, we do NOT implement `serde` traits.
```

File: design/src/rfcs/rfc0030_serialization_and_deserialization.md [hunk 4]
```diff
@@ -206,53 +206,56 @@ The resulting size of the serialized data is smaller when tagged externally, as
...
-Any fields with these data types are tagged with `#[serde(skip, default = "..name of the function for tailored serialization/de-serialization")]`.
+## Data Types to Skip Serialization/Deserialization
+We are going to skip serialization and deserialization of fields that have the datatype that corresponds to `@streaming blob` from smithy.
+Any fields with these data types are tagged with `#[serde(skip)]`.
 
-As of writing, this decision affects following Rust data types.
-- `aws_smithy_http::byte_stream::ByteStream`
-- `aws_smithy_http::event_stream::Receiver`
-- `aws_smithy_http::event_stream::EventStreamSender`
+By skipping, corresponding field's value will be assined the value generated by `Default` trait.
```
Comment (jdisanti) [inline design/src/rfcs/rfc0030_serialization_and_deserialization.md:213]:
```suggestion
By skipping, corresponding field's value will be assigned the value generated by `Default` trait.
```

**Review by jdisanti — COMMENTED** [review]
Some minor corrections. Otherwise, looks good!

**Comment by thomas-k-cameron** [comment]
@jdisanti 
Thank you for your suggestions!

**Review by jdisanti — APPROVED** [review]

**Comment by thomas-k-cameron** [comment]
Github says, `At least 6 approving reviews are required by reviewers with write access.`.

I feel something isn't working right.

**Comment by thomas-k-cameron** [comment]
Gotcha. Thanks!

**Pull request merged**